### DISCUSSION TITLE
RS485/Ethernet gateway by CQrobot

### DISCUSSION THREAD

**Discussion by rickyrickuk** [kellerza/sunsynk#190] [Q&A]
Hi, Just having my Sunsynk inverter installed and wondered if anyone has used a CQRobot RS485/Ethernet gateway?
On the specs it mentions it supports 'Modbus RTU/TCP bidirectional transparent transmission' but I cannot see a tickbox in the config pages like with other manufacturers.
More info on the gateway is here - https://www.cqrobot.com/index.php?route=product/product&product_id=1126
I have one already so was hoping it will be compatible.
Thanks

**Comment by kellerza** [comment]
This seems like it could work. Have a look at the settings on the mbusd addon if you are in doubt what the inverter expects

**Reply by rickyrickuk** [reply]
Thanks, I tried it as inverter was installed yesterday but just get timeout.
No where in the config utility to set modbus either 😒
I've ordered a waveshare unit to try as that has modbus settings.

**Reply by rickyrickuk** [reply]
In case anyone else wants to know, the Waveshare PoE device works a treat - https://www.waveshare.com/rs485-to-eth-b.htm?sku=23273
I went with the PoE one so no power supply needed.

**Reply by rickyrickuk** [reply]
Might have spoken too soon !
It worked for a day then stopped at 1am this morning.
I restarted the gateway and the ha app and it ran again, but has since stopped.
If I just restart the ha app then it tries to connect and gives timeout. Only way to fix is reboot the gateway.
